x-pack/plugin/mapper-flattened/src/main/java/org/elasticsearch/xpack/flattened/mapper/FlatObjectFieldMapper.java

```diff
@@ -319,6 +320,11 @@ public BytesRef indexedValueForSearch(Object value) {
             failIfNoDocValues();
             return new KeyedFlatObjectFieldData.Builder(name(), key, CoreValuesSourceType.BYTES);
         }
+
+        @Override
+        public ValueFetcher valueFetcher(MapperService mapperService, SearchLookup searchLookup, String format) {
+            throw new UnsupportedOperationException();  // TODO can we implement this?
```

Comment:
I was wondering the same, or even if we should do so. when would this field type be looked up compared to the other one that implements valueFetcher?

Reply:
This is a good question, I'll propose an idea in a follow-up PR.